schema/buildings/defs.yaml

```diff
@@ -0,0 +1,57 @@
+---
+"$schema": https://json-schema.org/draft/2020-12/schema
+title: Overture Maps Shared Building Properties
+description: Common schema definitions shared by building footprints and building parts
+shapeContainer:
+  title: Shape
+  description: Propeties of the buildings shape, such as height or roof type.
+  properties:
+    height:
+      description: >-
+        Height of the building or part in meters. The height is the distance from the lowest point to the highest point.
+      type: number
+      exclusiveMinimum: 0
+    numFloors:
+      description: >-
+        Number of above-ground floors of the building or part.
+      type: integer
+      exclusiveMinimum: 0
+    minHeight:
+      description: >-
+        The height of the bottom part of building in meters. Used if a building or part of building starts above the ground level.
+      type: number
+    roofShape:
+      description: The shape of the roof
+      type: string
+      enum:
+        - dome
+        - flat
+        - gable
+        - gambrel
+        - half-hip
+        - hip
+        - mansard
+        - onion
+        - pyramid
+        - round
+        - spherical
```

Comment:
What is the difference between `spherical` and `dome`? In osm there are only 25 instances of `roof:shape=spherical` and I think they are incorrectly tagged and can be replaced with `dome`.  Example: [way:945021143](https://www.openstreetmap.org/way/945021143)

Reply:
As discussed in the meeting, dome is something that is half a sphere. But sphere is needed to model something like Epcot or the new Sphere in Vegas. So we decided to keep it.